cloud-inquisitor/graph/schema.resolvers.go

```diff
@@ -15,7 +16,7 @@ func (r *accountResolver) Zones(ctx context.Context, obj *model.Account) ([]*mod
 	log.Infof("account <%v> getting zones\n", obj.AccountID)
 	log.Debugf("%#v\n", *obj)
 
-	db, err := NewDBConnection()
+	db, err := database.NewDBConnection()
```

Comment:
This isnt your issue; but can we fix this DB call? The resolver should be using `r.DB` instead of creating its own connection.